examples/bindgen_toolchain/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
+load("@bazel_skylib//rules:native_binary.bzl", "native_binary")
+load("@rules_rust_bindgen//:defs.bzl", "rust_bindgen_toolchain")
+
+native_binary(
+    name = "clang",
+    src = "@llvm_toolchain_llvm//:bin/clang",
+)
+
+cc_library(
+    name = "libclang",
+    srcs = ["@llvm_toolchain_llvm//:libclang"],
+)
+
+rust_bindgen_toolchain(
+    name = "my_bindgen_toolchain",
+    bindgen = "@rules_rust_bindgen//3rdparty:bindgen",
+    clang = ":clang",
+    libclang = ":libclang",
+)
+
+build_test(
```

Comment:
I think you wanna actually have a use of the toolchain in this example through toolchain resolution. Why not copy the simple test here instead?

Reply:
Using `bindgen_test_suite` here isn't straightforward because its deps are dev_dependencies of `@rules_rust_bindgen`. I used a symlink to reuse the integration test instead.

Reply:
What is a `bindgen_test_suite`?

Reply:
from https://github.com/bazelbuild/rules_rust/blob/82576d04b31a3123c824375385c3b2d5339da16d/extensions/bindgen/test/bindgen_test.bzl